static/src/javascripts/projects/common/modules/commercial/membership-engagement-banner.js

```diff
@@ -158,13 +158,15 @@ define([
         }
 
         function userHasMadeEnoughVisits(edition) {
+            const numberOfVisits = storage.local.get('gu.alreadyVisited') || 0;
             if (edition === 'INT') {
-                var internationalTestVariant = getVariant('MembershipEngagementInternationalExperiment');
-                if (internationalTestVariant == '1st_article')
-                    return true;
+                var internationalTestVariant = getVariant('MembershipEngagementInternationalExperimentTest12');
+                if (internationalTestVariant && internationalTestVariant !== 'control' && internationalTestVariant !== notInTest)
+                    //variants are in the form '1st_article', '3rd_article' so we can derive the number of visits from the name
+                    return numberOfVisits >= internationalTestVariant.substring(0, 1);
```

Comment:
Implicit conversions make me a little nervous. Should we `parseInt(internationalTestVariant.substring(0, 1))`